content/docs/concepts/remote_write_spec_2_0.md

```diff
@@ -0,0 +1,394 @@
+1. Noticing the target is no longer exposing the time series between successive scrapes
+1. Failing to scrape the target that originally exposed a time series
+1. Tracking configuration and evaluation for recording and alerting rules
+
+#### Metadata
+
+Metadata SHOULD follow the official guidelines for [TYPE](https://prometheus.io/docs/instrumenting/writing_exporters/#types) and [HELP](https://prometheus.io/docs/instrumenting/writing_exporters/#help-strings).
+
+#### Exemplars
+
+<!---
+TODO(bwplotka): Anything to say here?
+-->
+TBD
+
+#### Created Timestamp
+
+<!---
+TODO(bwplotka): TBD
+-->
+TBD
+
+### Responses
+
+Receiver ingesting all samples successfully MUST return HTTP 200 status code. In such a successful case, the response body from the Receiver SHOULD be empty; Sender MUST ignore the response body. The response body is RESERVED for future use.
+
+The following subsections specify Sender and Receiver semantics around write errors.
+
+#### Partial Write
+
+Sender SHOULD use Prometheus Remote Write to request write of multiple samples, across multiple series. As a result, Receiver MAY ingest valid samples within a write request that contains invalid or otherwise unwritten samples, which represents a partial write case.
+
+In a partial write case, Receiver MUST NOT return HTTP 200 status code. Receiver MUST provide a human-readable error message in the response body. Sender MUST NOT try and interpret the error message, and SHOULD log it as is.
+
+#### Unsupported Request Content
+
+Receiver MAY NOT support certain content types or encodings defined in [the Protocol section](#protocol). Receiver MUST return [415 HTTP Unsupported Media Type](https://www.rfc-editor.org/rfc/rfc9110.html#name-415-unsupported-media-type) status code if they don't support a given content type or encoding provided by the Sender.
+
+Sender SHOULD expect [400 HTTP Bad Request](https://www.rfc-editor.org/rfc/rfc9110.html#name-400-bad-request) for the above reasons from the 1.x Receiver, for backward compatibility.
+
+<!---
+TODO(bwplotka): Note sure if worth mentioning given we decided to not include auto negotiation logic in 2.0. I think I would delete it.
+-->
+Sender MAY retry write requests on 415 HTTP status code, with different content type and compression settings.
+
+#### Invalid Samples
+
+<!---
+TODO(bwplotka): This wording assumes metadata is optional, which I think it should NOT be.
+-->
+Receiver MAY NOT support certain metric types or samples (e.g. Receiver might reject sample without metadata or without created timestamp, while another Receiver might accept such sample.). It’s up to the Receiver what sample is invalid. Receiver MUST return a [400 HTTP Bad Request](https://www.rfc-editor.org/rfc/rfc9110.html#name-400-bad-request) status code for write requests that contain any invalid samples, unless the [partial retryable write](#retries-on-partial-writes) occurs.
```

Comment:
We should clarify, do we mean any invalid samples or all? I don't know for sure if any exists at the moment, but unless the spec clarifies that partial ingestion must be supported then it's possible that some systems are simply failing entire write requests if one sample is invalid?

In either case here, `the response SHOULD contain a useful human readable error about why samples were rejected`